parsec/backend/postgresql/pki.py

```diff
@@ -0,0 +1,439 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+import hashlib
+from typing import List
+from uuid import UUID
+from pendulum import DateTime
+
+from parsec.api.protocol import OrganizationID
+from parsec.api.protocol.pki import PkiEnrollmentStatus
+from parsec.api.protocol.types import UserID, UserProfile
+from parsec.backend.backend_events import BackendEvent
+from parsec.backend.postgresql.handler import send_signal
+from parsec.backend.user import UserActiveUsersLimitReached, UserAlreadyExistsError
+from parsec.backend.user_type import User, Device
+from parsec.backend.pki import (
+    PkiEnrollmentActiveUsersLimitReached,
+    PkiEnrollmentAlreadyEnrolledError,
+    PkiEnrollmentAlreadyExistError,
+    PkiEnrollmentCertificateAlreadySubmittedError,
+    PkiEnrollmentIdAlreadyUsedError,
+    PkiEnrollmentInfo,
+    PkiEnrollmentInfoAccepted,
+    PkiEnrollmentInfoCancelled,
+    PkiEnrollmentInfoRejected,
+    PkiEnrollmentInfoSubmitted,
+    PkiEnrollmentListItem,
+    BasePkiEnrollmentComponent,
+    PkiEnrollmentNoLongerAvailableError,
+    PkiEnrollmentNotFoundError,
+)
+from parsec.backend.postgresql import PGHandler
+from parsec.backend.postgresql.utils import Q, q_organization_internal_id, q_device_internal_id
+from parsec.backend.postgresql.user_queries.create import q_create_user
+
+_q_get_pki_enrollment_from_certificate_sha1 = Q(
+    f"""
+    SELECT * FROM pki_enrollment
```

Comment:
Should name the values instead of using `SELECT *` to avoid mixing fields with compatible value types